src/libraries/Common/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -369,7 +369,7 @@ private ValueTask SendFrameAsync(MessageOpcode opcode, bool endOfMessage, ReadOn
             // pass around (the CancellationTokenRegistration), so if it is cancelable, just immediately go to the fallback path.
             // Similarly, it should be rare that there are multiple outstanding calls to SendFrameAsync, but if there are, again
             // fall back to the fallback path.
-            return cancellationToken.CanBeCanceled || !_sendFrameAsyncLock.Wait(0) ?
+            return cancellationToken.CanBeCanceled || !_sendFrameAsyncLock.Wait(0, cancellationToken) ?
```

Comment:
We should instead pass `default` here to silence the warning. `0` is a special value that just polls, and we don't want this code path to throw for cancellation.